File: FizzBuzz.java
public class FizzBuzz {
    public static final void main(String argv[]){
        for(int i = 1; i <= 100; i++){
            System.out.println(FizzBuzz.fizzBuzz(i));
        }
    }

    public static String fizzBuzz(int number) {
        boolean fizz = (number % 3 ) == 0;
        boolean buzz = (number % 5 ) == 0;
        String result = "";

        if (fizz) result += "Fizz";
        if (buzz) result += "Buzz";
        if (!fizz && !buzz) result += String.valueOf(number);

        return result;
    }
}

File: fizzBuzz.js
var fizzBuzz = function (number) {
    var fizz = (number % 3 ) == 0;
    var buzz = (number % 5 ) == 0;
    var result = '';

    if (fizz) result += 'Fizz';
    if (buzz) result += 'Buzz';
    if (!fizz && !buzz) result += number;

    console.log (result);
}

for(let i = 1; i <= 100; i++){
    fizzBuzz(i);
}






